lib/mcstat.inc

```diff
@@ -98,47 +98,40 @@ function hr_time($seconds) {
  */
 class mcstat_client
 {
-  private $socket_ptr = null;
+  private $mc = null;
```

Comment:
Preference for this property name to be spelled out, unless it's adopting a convention from other code in the repo.

Reply:
Done.